pinger_bearing: 100.0
pinger_bearing_confidence: 0.0
pinger_range_confidence: 500.0
test_param: 400
test_param_2: 500
